packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -146,35 +167,88 @@ export class GCFBootstrapper {
     return config as Options;
   }
 
+  parseRequestHeaders(
```

Comment:
This logic already existed before - I just pulled it out into a separate function

Reply:
This can probably be static, private, or live outside the class completely.

Reply:
sure, I'll move it

Reply:
I've made them static + private but kept them in the class since they're only used within it